### PR TITLE
NTT-108 | Initial service setup

### PR DESCRIPTION
**Pull request by owen-opencast**

### CODE REVIEW

File: app/navigation/Navigator.scala [hunk 1]
```diff
@@ -27,7 +27,15 @@ import models._
 class Navigator @Inject()() {
 
   private val normalRoutes: Page => UserAnswers => Call = {
-    case _ => _ => routes.IndexController.onPageLoad()
+    case  WasTrustSetUpAfterSettlorDiedPage => _ => routes.WhatIsSettlorsNameController.onPageLoad(NormalMode)
```
Comment (ajchilts) [inline app/navigation/Navigator.scala]: Maybe align the => _ => for neatness?

File: conf/app.routes [hunk 2]
```diff
@@ -13,3 +13,58 @@ GET        /check-your-answers                          controllers.CheckYourAns
 GET        /unauthorised                                controllers.UnauthorisedController.onPageLoad
 
 GET        /accessibility-statement                     controllers.AccessibilityController.onPageLoad
+
+GET        /doYouKnowCountryOfNationality                        controllers.DoYouKnowCountryOfNationalityController.onPageLoad(mode: Mode = NormalMode)
```
Comment (ajchilts) [inline conf/app.routes]: Can we please correctly align the contents of this file for readability?

File: app/navigation/Navigator.scala (hunk 1 above)
Comment (owen-opencast) [reply app/navigation/Navigator.scala]: Fixed

File: conf/app.routes (hunk 2 above)
Comment (owen-opencast) [reply conf/app.routes]: I think this came from an earlier commit, not sure how!

File: app/controllers/livingsettlor/WhatIsCountryOfHeadOfficeController.scala [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+  def onSubmit(mode: Mode): Action[AnyContent] = (identify andThen getData andThen requireData).async {
+    implicit request =>
+
+      form.bindFromRequest().fold(
+        formWithErrors => {
+
+          val json = Json.obj(
+            "form" -> formWithErrors,
+            "mode" -> mode
```
[the remark this answers is not in the document]
Comment (owen-opencast) [reply app/controllers/livingsettlor/WhatIsCountryOfHeadOfficeController.scala:79]: 👍 Just missing in that file

File: app/controllers/livingsettlor/DoYouKnowCountryOfHeadOfficeController.scala [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import renderer.Renderer
+import repositories.SessionRepository
+import uk.gov.hmrc.play.bootstrap.controller.FrontendBaseController
+import uk.gov.hmrc.viewmodels.{NunjucksSupport, Radios}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class DoYouKnowCountryOfHeadOfficeController @Inject()(
+    override val messagesApi: MessagesApi,
```
[the remark this answers is not in the document]
Comment (owen-opencast) [reply app/controllers/livingsettlor/DoYouKnowCountryOfHeadOfficeController.scala:36]: Indentation was as generated by the scaffold?

File: app/controllers/livingsettlor/IndexController.scala [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import renderer.Renderer
+import repositories.SessionRepository
+import uk.gov.hmrc.play.bootstrap.controller.FrontendBaseController
+
+import scala.concurrent.ExecutionContext
+
+class IndexController @Inject()(
+    val controllerComponents: MessagesControllerComponents,
```
[the remark this answers is not in the document]
Comment (owen-opencast) [reply app/controllers/livingsettlor/IndexController.scala:33]: As above?

File: app/controllers/livingsettlor/DoYouKnowCountryOfHeadOfficeController.scala (hunk 4 above)
[the remark this answers is not in the document]
Comment (parkeradam) [reply app/controllers/livingsettlor/DoYouKnowCountryOfHeadOfficeController.scala:36]: Yeah but I think we said we were gonna indent to the first bracket. Apparently thats what normal, I don't like it but we should be consistent in the codebases.

[the remark this answers is not in the document]
Comment (owen-opencast) [reply app/controllers/livingsettlor/DoYouKnowCountryOfHeadOfficeController.scala]: Updated

File: app/controllers/livingsettlor/IndexController.scala (hunk 5 above)
[the remark this answers is not in the document]
Comment (owen-opencast) [reply app/controllers/livingsettlor/IndexController.scala]: Updated

**Pull request merged**